<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lqSaveLASDialog</class>
 <widget class="QDialog" name="lqSaveLASDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>258</width>
    <height>135</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select LAS export Parameters</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
  <item>
    <layout class="QHBoxLayout" name="horizontalLayoutExportType">
      <item>
        <widget class="QLabel" name="labelExportType">
          <property name="text">
            <string>Export Type</string>
          </property>
        </widget>
      </item>
      <item>>
        <widget class="QComboBox" name="ExportTypeComboBox"/>
      </item>
   </layout>
  </item>
  <item>
    <layout class="QHBoxLayout" name="horizontalLayoutUTMZone">
      <item>
        <widget class="QLabel" name="labelUTMZone">
          <property name="text">
           <string>UTMZone</string>
          </property>
        </widget>
      </item>
      <item>
      <widget class="QSpinBox" name="UTMZoneSpinBox">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>60</number>
       </property>
      </widget>
      </item>
    </layout>
  </item>
  <item>
    <widget class="QCheckBox" name="WriteColorCheckBox">
      <property name="text">
       <string>Write Color</string>
      </property>
    </widget>
  </item>
  <item>
    <widget class="QCheckBox" name="WriteSRSCheckBox">
      <property name="text">
       <string>Write SRS</string>
      </property>
    </widget>
  </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>lqSaveLASDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>lqSaveLASDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
